tests/global_config_arm64.py

```diff
@@ -1,20 +1,51 @@
+import os
 from typing import Any
 
 import pytest_testconfig
+from ocp_resources.template import Template
 
 from utilities.constants import (
     ARM_64,
+    DV_SIZE_STR,
     EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS,
+    FLAVOR_STR,
+    IMAGE_NAME_STR,
+    IMAGE_PATH_STR,
+    LATEST_RELEASE_STR,
+    OS_STR,
+    OS_VERSION_STR,
     PREFERENCE_STR,
+    TEMPLATE_LABELS_STR,
+    WORKLOAD_STR,
     Images,
 )
+from utilities.infra import get_latest_os_dict_list
 
 global config
 global_config = pytest_testconfig.load_python(py_file="tests/global_config.py", encoding="utf-8")
 
 Images.Cirros.RAW_IMG_XZ = "cirros-0.4.0-aarch64-disk.raw.xz"
 EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS[PREFERENCE_STR] = f"rhel.9.{ARM_64}"
 
+rhel_os_matrix = [
+    {
+        "rhel-9-5": {
```

Comment:
maybe rhel 9.6?

Reply:
9.6 is not yet GA. this will be updated when we get the GA images

Reply:
@vsibirsk ack rhel 9.6 will GA in May before us. so it will change before we release 4.19?